### DISCUSSION TITLE
Welcome to PUMA Discussions!

### DISCUSSION THREAD

**Discussion by LalithShiyam** [ENHANCE-PET/PUMA#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Abbaun** [comment]
I am a PhD student working with multimodal PET brain imaging (and MRI). I am intrigued by the capabilities of PUMA to visualize the complex interplay between modalities and I was wondering whether the algorithm would work on brain images or if it only works on whole body images?

**Reply by LalithShiyam** [reply]
Hi @Abbaun,

Many thanks for your interest! I am quite positive it will work on brain as well. A few tweaks might be needed. I am currently at LMU and this of interest to them as well. So I might work on implementing this. What do you have in mind?

Cheers,
Lalith

**Reply by Abbaun** [reply]
I have 3 PET modalities that I would like to visualize the colocalisation of in the brain. A dopaminergic tracer, a cholinergic tracer and a tracer for neuroinflammation. So being able to visualise whether there is greater overlap between the dopaminergic tracer and inflammation, for instance, compared to the cholinergic tracer would be very interesting to me. 
Also, I am curious if it can be modified to work with MRI instead of CT as the anatomical reference?

**Reply by LalithShiyam** [reply]
Right, are they all PET/MR datasets? Meaning all the three datasets that you want to align?

**Reply by Abbaun** [reply]
It is actually 3 HRRT PET scans and separate MRI scans. So I am doing some initial preprocessing to coregister the images.
I have done a test run on a single subject. I was initially not able to get Puma to proper align the images for me, but that might be because of different origo in the PET and MRI scans. When I coregistered the images beforehand, I was able to make the algorithm work.

**Reply by LalithShiyam** [reply]
Alright, good to know - do you still want to include a 'smoother' way in PUMA or you are fine with what you have now?

**Reply by Abbaun** [reply]
It would be great with a smoother more plug and play solution for my type of data as well, but as long as it works, even with a few extra steps from my end, I am happy ;)
